repository-data/application/src/main/resources/hcm-config/namespaces/hee/expanderPanel.yaml

```diff
@@ -69,4 +76,17 @@ definitions:
             wicket.id: ${cluster.id}.field
             /cluster.options:
               jcr:primaryType: frontend:pluginconfig
-              ckeditor.config.appended.json: '{ removeButtons: "PickImage", forcePasteAsPlainText: true }'
+              ckeditor.config.appended.json: '{ removeButtons: "PickImage,Table",
```

Comment:
I think `ckeditor.config.appended.json` config isn't needed here as it would be inherited from `/hippo:namespaces/hippostd/html/editor:templates/_default_/@ckeditor.config.appended.json`.